appveyor.yml

```diff
@@ -11,12 +11,8 @@ environment:
     - RuntimeURL: https://atom.io/download/atom-shell
       RuntimeName: iojs
       RuntimeVersion: v2.0.8
+      NodeVersion: 8
```

Comment:
Why changing the node version? Also we should maybe use the LTS version.

Reply:
The iojs version v2.0.8 requires the node version 8 to run. Changed the node version to prevent any incompatibilities. Previously we were using version 9.